cps/editbooks.py

```diff
@@ -381,96 +381,89 @@ def move_mediafile(requested_files, current_user_name=None, shelf_title=None):
                 log.error("Error occurred: %s", ex)
                 flash(_("There was an error"), category="error")
 
-        log.info("Requested files list: %s", requested_files)
-        for requested_file in requested_files:
```

Comment:
Previously, we expected a list of requested files. Now files are processed in a case by case basis. Those files are sent from the TaskDownload from /cps/tasks.download.py. This is the reason here a block of code has been indented left, eliminating the previous for loop.